pkg/util/helper/unstructured.go

```diff
@@ -228,3 +229,20 @@ func ToUnstructured(obj interface{}) (*unstructured.Unstructured, error) {
 	}
 	return &unstructured.Unstructured{Object: uncastObj}, nil
 }
+
+// HasObjUpdated judges whether the object has updated.
+func HasObjUpdated(oldObj interface{}, newObj interface{}) bool {
+	oldMetaInfo, err := meta.Accessor(oldObj)
+	if err != nil { //should not happen
+		return false
+	}
+	newMetaInfo, err := meta.Accessor(newObj)
+	if err != nil { //should not happen
+		return false
+	}
+
+	if oldMetaInfo.GetResourceVersion() != newMetaInfo.GetResourceVersion() {
+		return true
+	}
```

Comment:
Looks weird that `.metadata.ResourceVersion` is not changed in an update event.
I doubt it.